.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -42,3 +45,9 @@ jobs:
         with:
           name: protractor-screenshots
           path: /home/runner/work/oppia/protractor-screenshots
+      - name: Uploading protractor-video as Artifacts
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false && always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: protractor-video
+          path: /home/runner/work/oppia/oppia/core/protractor-video
```

Comment:
Maybe try moving the `protractor-video` folder out of `oppia/core`? We should probably keep it under `/home/runner/work/oppia/` like `protractor-screenshots`

Reply:
moved!